src/Raven.Server/Documents/ChangesClientConnection.cs

```diff
@@ -552,7 +580,7 @@ await using (var ms = new MemoryStream())
                         context.Renew();
 
                         var messagesCount = 0;
-                        await using (var writer = new AsyncBlittableJsonTextWriter(context, ms))
+                        using (var writer = new BlittableJsonTextWriter(context, ms))
```

Comment:
why?

Reply:
We are writing here to a memory stream, which is never actually async.
We don't need any of the async machinery here